src/zenhub.py

```diff
@@ -95,13 +181,13 @@ def _update_issue_pipeline(self, pipeline, pos=None):
         if pipeline in self.pipeline_ids:
             logger.info(f'Changing the current value of pipeline to {pipeline}')
 
-            url = os.path.join(self.url, 'moves')
             json_dict = {'pipeline_id': self.pipeline_ids[pipeline], 'position': pos or 'top'}
 
-            response = requests.post(url, headers=self.headers, json=json_dict)
+            response = requests.post(f'{self.url}{self.repo_id}/issues/{self.github_key}/moves', headers=self.headers,
```

Comment:
For the sake of consistency, I would choose either `r` or `response` as variable names of an HTTP response. I've seen both.

Reply:
switched all usages to r